cmd/get.go

```diff
@@ -54,21 +68,44 @@ func getCMDsFlags(getCmd *cobra.Command) {
 	getCmd.PersistentFlags().Int("max-content-length", 0, "Max content length in MB to download for a single resource.")
 	getCmd.PersistentFlags().Float64("min-space-required", 0, "Minimum space required in GB to continue the crawl. Default will be 50GB * (total disk space / 256GB) if total disk space is less than 256GB, else 50GB.")
 	getCmd.PersistentFlags().Bool("strict-regex", false, "If turned on, the xurls `strict` regex setting will be used. Otherwise a looser regex will be used.")
+}
+
+func addHeadlessFlags(getCmd *cobra.Command) {
+	getCmd.PersistentFlags().Bool("headless", false, "[headless] Run in headless mode (experimental).")
+	getCmd.PersistentFlags().Bool("headless-headful", false, "[headless] Switch to headful mode to run the browser with full graphics stack and show browser GUI (requires --headless).")
+	getCmd.PersistentFlags().Bool("headless-trace", false, "[headless] Trace enables the visual tracing of the input actions on the page.")
+	getCmd.PersistentFlags().Bool("headless-dev-tools", false, "[headless] F12")
+	getCmd.PersistentFlags().Bool("headless-stealth", false, "[headless] Use stealth to prevent bot detection and use the browser's native User-Agent")
+	getCmd.PersistentFlags().Bool("headless-user-mode", false, "[headless] Launch your regular user browser session")
+
+	getCmd.PersistentFlags().Int("headless-chromium-revision", headless.DefaultChromiumRevision, "[headless] Default Chromium binary revision to auto download and use (if --headless-chromium-bin is not set). set to -1 to query the latest revision from chromiumdash.appspot.com")
```

Comment:
I think I would want to see this the other way round - set to -1 by default to ensure we are getting the latest version (with the "default" binary in the description)

Reply:
Done.

Note that the Chromium stable channel is released every week.
we probably need to clean up `~/.cache/rod/browser/` from time to time.